src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -219,7 +221,11 @@
   [b]
   (if (instance? Boolean b)
     b
-    (Boolean/parseBoolean b)))
+    (case b
+      "true" true
```

Comment:
I don't understand why this change was needed, was this a fix for something? This changes the behavior from what was there before. parseBoolean will ignore case and accept True as an example. It will also accept any string, returning false for things that don't fit.